REB_firmware/REB_retrieve_versions_after/v0/ccseoREB_retrieve_versions.py

```diff
@@ -12,14 +12,17 @@
 
 cdir = tsCWD
 
-
```

Comment:
I have asked our eTraveler person here at BNL about this and he has requested to keep them separate as he believes it would be more clear to our operators to have a specific step verifying the upgrade than if it is bundled with another step (some of our more non-technical operators really need to have eTravelers clearly separated into parts to avoid confusion).